src/networks.json

```diff
@@ -1172,6 +1172,20 @@
     "start": 1715902,
     "logo": "ipfs://QmTogMDLmDgJjDjUKDHDuc2KVTVDbXf8bXJLFiVe8PRxgo"
   },
+  "53935": {
+    "key": "53935",
+    "name": "DFK Chain",
+    "chainId": 53935,
+    "network": "mainnet",
+    "multicall": "0x5b24224dC16508DAD755756639E420817DD4c99E",
```

Comment:
Hi @friskyfoxdk Need a way to verify multicall code through explorer, make sure it is verified and published to explorer

https://subnets.avax.network/defi-kingdoms/address/0x5b24224dC16508DAD755756639E420817DD4c99E?transfers=code

Reply:
Sorry for the delay in this, it's been an effort to get the validation working for this, but we got it: https://subnets.avax.network/defi-kingdoms/address/0x5b24224dC16508DAD755756639E420817DD4c99E?transfers=code